src/test/java/seedu/address/logic/parser/EditCommandParserTest.java

```diff
@@ -77,59 +77,59 @@ public void parse_invalidPreamble_failure() {
     @Test
     public void parse_invalidValue_failure() {
         assertParseFailure(parser, " spot/Test" + INVALID_NAME_DESC, Name.MESSAGE_CONSTRAINTS); // invalid name
-        assertParseFailure(parser, " spot/Test" + INVALID_RATING_DESC, Rating.MESSAGE_CONSTRAINTS); // invalid phone
+        assertParseFailure(parser, " spot/Test" + INVALID_RATING_DESC, Rating.MESSAGE_CONSTRAINTS); // invalid rating
         //  No invalid emails
         //  assertParseFailure(parser, " spot/Test" + INVALID_EMAIL_DESC, Email.MESSAGE_CONSTRAINTS); // invalid email
         assertParseFailure(parser, " spot/Test" + INVALID_ADDRESS_DESC,
                 Address.MESSAGE_CONSTRAINTS); // invalid address
         assertParseFailure(parser, " spot/Test" + INVALID_TAG_DESC, Tag.MESSAGE_CONSTRAINTS); // invalid tag
 
-        // invalid phone followed by valid email
-        assertParseFailure(parser, " spot/Test" + INVALID_RATING_DESC + EMAIL_DESC_AMY,
+        // invalid rating followed by valid email
+        assertParseFailure(parser, " spot/Test" + INVALID_RATING_DESC + EMAIL_DESC_FRONTIER,
```

Comment:
One thing we could perhaps do is abstract out the " spot/Test" into a VALID_EDIT_DESC